File: exercise/java/day24/src/test/java/delivery/Time.java
package delivery;

import java.time.LocalDateTime;
import java.time.Month;
import java.time.ZoneOffset;
import java.util.function.Supplier;

public class Time {
    public static final LocalDateTime NOW = LocalDateTime.of(2024, Month.DECEMBER, 24, 23, 30, 45).atOffset(ZoneOffset.UTC).toLocalDateTime();
    public static final Supplier<LocalDateTime> PROVIDER = () -> NOW;
}

File: exercise/php/day24/tests/Time.php
<?php

namespace Tests;

use DateTime;
use DateTimeZone;

class Time
{
    public static DateTime $NOW;
    public static $PROVIDER;

    /**
     * @throws \Exception
     */
    public static function init(): void
    {
        self::$NOW = new DateTime('2024-12-24 23:30:45', new DateTimeZone('UTC'));
        self::$PROVIDER = function () {
            return self::$NOW;
        };
    }
}

Time::init();
